build/build-rector-scoped.sh

```diff
@@ -37,14 +37,12 @@ note "Running scoper to $RESULT_DIRECTORY"
 wget https://github.com/humbug/php-scoper/releases/download/0.14.0/php-scoper.phar -N --no-verbose
 
 # Work around possible PHP memory limits
-php -d memory_limit=-1 php-scoper.phar add-prefix preload.php bin config src packages rules vendor composer.json --output-dir "../$RESULT_DIRECTORY" --config scoper.php --force --ansi --working-dir "$BUILD_DIRECTORY"
+php -d memory_limit=-1 php-scoper.phar add-prefix bin config src packages rules vendor composer.json --output-dir "../$RESULT_DIRECTORY" --config scoper.php --force --ansi --working-dir "$BUILD_DIRECTORY"
```

Comment:
I think this would remove the `preload.php` from copied files

Reply:
Fixed at https://github.com/rectorphp/rector-src/pull/592/commits/7810227f2c6fa41cf1c7872b1db9b2415abfa6f2 for use copy instead of scoping it.

Reply:
Yea, that makes sense :+1: